### PR TITLE
Replaced WebL10n to i18next in Chat Prototype

### PR DESCRIPTION
**Pull request by Vishalk91-4**
Replaced WebL10n in Chat Prototype Activity
All tutorials working in the languages mentioned in Chat Prototype activity

image
@llaske, could you please review it

### CODE REVIEW

File: activities/ChatPrototype.activity/lib/l10n.js
Comment (llaske) [inline activities/ChatPrototype.activity/lib/l10n.js:1]: The code of this file has been updated [here](https://github.com/llaske/sugarizer/blob/dev/lib/l10n.js).
Could you update your PR with the recent version?

**Comment by Vishalk91-4** [comment]
@lllaske, updated PR with latest l10n.js

File: activities/ChatPrototype.activity/lib/sugar-web/env.js
Comment (llaske) [inline activities/ChatPrototype.activity/lib/sugar-web/env.js:1]: This file is not supposed to be updated.

Comment (Vishalk91-4) [reply activities/ChatPrototype.activity/lib/sugar-web/env.js:1]: Yes, Changed it back to original

**Comment by llaske** [comment]
There is an error when sharing the activity:

image

**Comment by Vishalk91-4** [comment]
@llaske , Changed all of those now

**Comment by llaske** [comment]
Works now. Thanks.

**Comment by llaske** [comment]
Add reference to https://github.com/llaske/sugarizer/issues/1378

**Pull request merged**